### PR TITLE
Added CVE-2019-19346

### PR DESCRIPTION
**Pull request by tausif-rh**

### CODE REVIEW

File: 2019/19xxx/CVE-2019-19346.json [hunk 1]
```diff
@@ -4,15 +4,68 @@
...
+                {
+                    "vendor_name": "Openshift Enterprise",
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "openshift/mariadb-apb",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_value": "n/a"
```
Comment (cve-team) [inline 2019/19xxx/CVE-2019-19346.json]: CVE entries must contain version information ([8.1.1](https://cve.mitre.org/cve/cna/rules.html#section_8-1_cve_entry_information_requirements))

Comment (tausif-rh) [reply 2019/19xxx/CVE-2019-19346.json]: Added version information.

**Comment by cve-team** [comment]
Also, be sure to add any version information to the description, thanks!

File: 2019/19xxx/CVE-2019-19346.json [hunk 2]
```diff
@@ -4,15 +4,78 @@
...
+                "refsource": "CONFIRM"
+            }
+        ]
     },
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "An insecure modification vulnerability in the /etc/passwd file was found in the container openshift/mariadb-apb. An attacker with access to the container could use this flaw to modify /etc/passwd and escalate their privileges."
```
Comment (cve-team) [inline 2019/19xxx/CVE-2019-19346.json]: Missing affected versions in the description.

**Comment by tausif-rh** [comment]
> The description says that versions before 4.3.5, 4.2.21, 4.1.37, and 3.11.188-4 are affected. However, the version_data field says 4.3.5-202003020549, 4.2.21-202002240343, 4.1.37-202003021622, and 3.11.188-4 are affected. This doesn't add up. Please verify and make the proper changes.

@cristina479 
The difference is the date part of the version number. The extra date information from the version number was removed for the description to be cleaner. Do you wish to have that information there ?

**Comment by cristina479** [comment]
> > The description says that versions before 4.3.5, 4.2.21, 4.1.37, and 3.11.188-4 are affected. However, the version_data field says 4.3.5-202003020549, 4.2.21-202002240343, 4.1.37-202003021622, and 3.11.188-4 are affected. This doesn't add up. Please verify and make the proper changes.
> 
> @cristina479
> The difference is the date part of the version number. The extra date information from the version number was removed for the description to be cleaner. Do you wish to have that information there ?

I'm not concerned about the date part. The **before** is what makes the versions in the description and in the version_data to differ. The version_data field should say before 4.3.5-202003020549, before 4.2.21-202002240343, before 4.1.37-202003021622, and before 3.11.188-4.

**Comment by tausif-rh** [comment]
> > > The description says that versions before 4.3.5, 4.2.21, 4.1.37, and 3.11.188-4 are affected. However, the version_data field says 4.3.5-202003020549, 4.2.21-202002240343, 4.1.37-202003021622, and 3.11.188-4 are affected. This doesn't add up. Please verify and make the proper changes.
> > 
> > 
> > @cristina479
> > The difference is the date part of the version number. The extra date information from the version number was removed for the description to be cleaner. Do you wish to have that information there ?
> 
> I'm not concerned about the date part. The **before** is what makes the versions in the description and in the version_data to differ. The version_data field should say before 4.3.5-202003020549, before 4.2.21-202002240343, before 4.1.37-202003021622, and before 3.11.188-4.

They're not the same. The version_data that we have provided is the "Fixed In" versions of the package. While that in the description is about the "Affected" Versions of the package.

**Comment by cristina479** [comment]
> > > > The description says that versions before 4.3.5, 4.2.21, 4.1.37, and 3.11.188-4 are affected. However, the version_data field says 4.3.5-202003020549, 4.2.21-202002240343, 4.1.37-202003021622, and 3.11.188-4 are affected. This doesn't add up. Please verify and make the proper changes.
> > > 
> > > 
> > > @cristina479
> > > The difference is the date part of the version number. The extra date information from the version number was removed for the description to be cleaner. Do you wish to have that information there ?
> > 
> > 
> > I'm not concerned about the date part. The **before** is what makes the versions in the description and in the version_data to differ. The version_data field should say before 4.3.5-202003020549, before 4.2.21-202002240343, before 4.1.37-202003021622, and before 3.11.188-4.
> 
> They're not the same. The version_data that we have provided is the "Fixed In" versions of the package. While that in the description is about the "Affected" Versions of the package.

The version_data field is inside the "affects" object. Currently it reads as if 4.3.5-202003020549, 4.2.21-202002240343, 4.1.37-202003021622, and 3.11.188-4 are affected. To indicate that these versions are the "fixed in" version, the version_data field should say something like "Fixed in 4.3.5-202003020549, Fixed in 4.2.21-202002240343, Fixed in 4.1.37-202003021622, and Fixed in 3.11.188-4".

**Comment by tausif-rh** [comment]
> > > > > The description says that versions before 4.3.5, 4.2.21, 4.1.37, and 3.11.188-4 are affected. However, the version_data field says 4.3.5-202003020549, 4.2.21-202002240343, 4.1.37-202003021622, and 3.11.188-4 are affected. This doesn't add up. Please verify and make the proper changes.
> > > > 
> > > > 
> > > > @cristina479
> > > > The difference is the date part of the version number. The extra date information from the version number was removed for the description to be cleaner. Do you wish to have that information there ?
> > > 
> > > 
> > > I'm not concerned about the date part. The **before** is what makes the versions in the description and in the version_data to differ. The version_data field should say before 4.3.5-202003020549, before 4.2.21-202002240343, before 4.1.37-202003021622, and before 3.11.188-4.
> > 
> > 
> > They're not the same. The version_data that we have provided is the "Fixed In" versions of the package. While that in the description is about the "Affected" Versions of the package.
> 
> The version_data field is inside the "affects" object. Currently it reads as if 4.3.5-202003020549, 4.2.21-202002240343, 4.1.37-202003021622, and 3.11.188-4 are affected. To indicate that these versions are the "fixed in" version, the version_data field should say something like "Fixed in 4.3.5-202003020549, Fixed in 4.2.21-202002240343, Fixed in 4.1.37-202003021622, and Fixed in 3.11.188-4".

Added. RFR

**Pull request merged**